repos/system_upgrade/el7toel8/actors/addupgradebootentry/libraries/library.py

```diff
@@ -12,12 +12,12 @@ def add_boot_entry():
     kernel_dst_path, initram_dst_path = get_boot_file_paths()
     run([
         '/usr/sbin/grubby',
-        '--add-kernel={0}'.format(kernel_dst_path),
-        '--initrd={0}'.format(initram_dst_path),
-        '--title="RHEL Upgrade Initramfs"',
+        '--add-kernel', '{0}'.format(kernel_dst_path),
+        '--initrd', '{0}'.format(initram_dst_path),
+        '--title', 'RHEL Upgrade Initramfs',
```

Comment:
```suggestion
        '--title', '"RHEL Upgrade Initramfs"',
```
I would leave there the quotation marks, the same as you would use them in a terminal.
Neither stdlib _run_ or __call_ tell in description how the processing of the list works, so I'd better stay defensive.
CCing @shaded-enmity.

Reply:
That's not defensive; you're just passing the text *with* quotes.

I'm not sure about `run` and `_call` but standard library passes the args right to the forked process without involving shell at all.

Reply:
@bocekm @AloisMahdal The list of arguments is passed verbatim to `os.exec*` function.

Reply:
I.e. 
```
os.exec('/bin/echo', ['/bin/echo', '"FOO"'])
"FOO" 
```

whereas

```
os.exec('/bin/echo', ['/bin/echo', 'FOO'])
FOO
```

So it depends on what Grubby expects.

Reply:
Yep, that's what I expected.  (It's the safest way to call commands anyway...)

So quotes/spaces/any-old-junk does not really influence "safety".